packages/api/internal/dns/server.go

```diff
@@ -38,11 +39,15 @@ type DNS struct {
 	}
 }
 
-func New(ctx context.Context, rc *redis.Client) *DNS {
+func New(ctx context.Context, rc *redis.Client, rc2 *redis.Client) *DNS {
```

Comment:
why are there two Redis clients at the same time now? I am confused

Reply:
As described in description, this way we can populate data without causing downtime, if the redis restarts (or if we replace it with a new one, it would lost all routing information to sandboxes, all sandboxes started prior to the change would be inaccessible).

After 24 hours we will be sure that all data are in redis and we can remove this and use only the new redis without any issue